src/Components/Publishing/Nav/Nav.tsx

```diff
@@ -41,39 +41,21 @@ export class NavComponent extends React.Component<Props, State> {
   }
 
   render() {
-    const {
-      sponsor,
-      className,
-      canFix,
-      transparent,
-      title
-    } = this.props
+    const { sponsor, className, canFix, transparent, title } = this.props
     const { isFixed } = this.state
 
     return (
       <div>
-        <NavContainer
-          transparent={!isFixed && transparent}
-          className={className}
-          isFixed={canFix && isFixed}
-        >
```

Comment:
😢 bye bye readable code

Reply:
😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢 😢

Reply:
I don't really get this, the new code looks better to me. 

It's short enough to fit on a single line, so it becomes more readable by being read left to right?

Reply:
Yeah I guess it’s just personal preference - i like Prettier for everything except being able to do multiline components like this. We do this consistently in the Publishing directory and in Positron so I think we’re just used to it. But I can see how this is troublesome within the repo for half of us to use it and the other half not if we want to share components for instance!

Reply:
That feels like the answer is to get prettier installed and set up the same in Positron too 👍